modules/templates/templates_api.rb

```diff
@@ -14,15 +14,21 @@ class Proxy::TemplatesApi < Sinatra::Base
     end
   end
 
+  get "/userdata/:kind" do |kind|
+    log_halt(500, "Failed to retrieve #{kind} userdata template for #{params.inspect}: ") do
+      Proxy::Templates::TemplateProxyRequest.new.get_template(['userdata', kind].join('/'), request.env, params)
```

Comment:
It looks like all of these join an array and inside the function it's split into an array. Shouldn't the get_template function accept an array instead?